whelk-core/src/main/groovy/whelk/JsonLd.groovy

```diff
@@ -1153,3 +1178,104 @@ class JsonLd {
         }
     }
 }
+
+
+@Immutable
+class Link {
+    String iri
+    String relation
+
+    Link withIri(String iri) {
+        iri == this.iri
+                ? this
+                : new Link(iri: iri, relation: this.relation)
+    }
+}
+
+class Embellisher {
```

Comment:
Nice refactoring out! If we're using these outside of the class in the same file, it's good to have a distinct file for them.

Reply:
:+1: